src/Neo/SmartContract/Native/PolicyContract.cs

```diff
@@ -418,5 +418,16 @@ private bool UnblockAccount(ApplicationEngine engine, UInt160 account)
             engine.SnapshotCache.Delete(key);
             return true;
         }
+
+        [ContractMethod(Hardfork.HF_Faun, CpuFee = 1 << 15, RequiredCallFlags = CallFlags.ReadStates)]
+        private StorageIterator GetBlockedAccounts(ApplicationEngine engine)
```

Comment:
We already have `IsBlocked` method, so we may consider renaming it to `GetBlocked` for naming consistency.